lfucg-exactions/client/js/components/AccountLedgerForm.js

```diff
@@ -101,7 +111,7 @@ class AccountLedgerForm extends React.Component {
                                 <fieldset>
                                     <div className="row">
                                         <div className="col-sm-6 form-group">
-                                            <label htmlFor="lot_id" className="form-label" id="lot_id">Lot</label>
+                                            <label htmlFor="lot_id" className="form-label" id="lot_id" aria-label="Required Lot">* Lot</label>
```

Comment:
you can use aria-required="true" to tell screen readers that this field is required.